doc/sphinx/changes.rst

```diff
@@ -14,6 +14,134 @@ Version 9.0
 Summary of changes
 ~~~~~~~~~~~~~~~~~~
 
+The Rocq Prover version 9.0 is the first Rocq Prover release after the renaming
+from The Coq Proof Assistant. The Rocq Prover 9.0 is backwards compatible with
+Coq 8.20, providing compatitibility shims so that developments depending on Coq
+can be easily ported, see `Porting to The Rocq Prover`_  for details. The 9.0 version
+is based on a new single binary `rocq` that dispatches commands to previously
+separate binaries, a split and renaming of the standard library to `Stdlib` and
+an internal encoding of template-polymorphism in terms of the
+more general notion of sort polymorphism.
```

Comment:
Strictly speaking tmplate poly in 9.0 is not internally encoded in terms of sort poly, rather its theory was rethought in terms of sort poly and the implementation changed to be closer (specifically to remove the extra generated constraints). PRs like https://github.com/coq/coq/pull/19985 which actually implement it in terms of sort poly are for 9.1.

Reply:
Fixed